core/src/main/scala/org/apache/spark/scheduler/InputFormatInfo.scala

```diff
@@ -162,6 +163,7 @@ object InputFormatInfo {
     go to (a) until required nodes are allocated.
 
     If a node 'dies', follow same procedure.
+    }}}
```

Comment:
- Java
  <img width="629" alt="2016-11-27 3 17 20" src="https://cloud.githubusercontent.com/assets/6477701/20642361/12bc2eac-b450-11e6-88c3-80a6f4b9a449.png">

Scaladoc not found.